components/react/hooks/src/useScroll/index.js

```diff
@@ -0,0 +1,43 @@
+import {useState, useEffect} from 'react'
+
+export default function useScroll({axis = 'Y'} = {}) {
+  const getBounding = () =>
```

Comment:
I'd rather flip the condition here and use:
- a more specific naming
- optional chaining

```javascript

const getBoundingClientRect = () => 
   window?.document
        ? document.body.getBoundingClientRect()
        : 0 
```
I hope this returns the same result you are expecting 😃 !

Reply:
PR updated, thank you @davecarter  for your suggestion 😊